src/PInvoke.sln.DotSettings

```diff
@@ -64,6 +67,27 @@
+	<s:String x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/CustomProperties/=Extension/@EntryIndexedValue">cs</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/CustomProperties/=FileName/@EntryIndexedValue">Base+Nested</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/CustomProperties/=ValidateFileName/@EntryIndexedValue">False</s:String>
+	<s:String x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Description/@EntryValue">Nested PInvoke declaration</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Field/=BASECLASS/@KeyIndexDefined">True</s:Boolean>
+	<s:Int64 x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Field/=BASECLASS/Order/@EntryValue">0</s:Int64>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Field/=NESTED/@KeyIndexDefined">True</s:Boolean>
+	<s:Int64 x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Field/=NESTED/InitialRange/@EntryValue">1</s:Int64>
+	<s:Int64 x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Field/=NESTED/Order/@EntryValue">1</s:Int64>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Reformat/@EntryValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Scope/=E8F0594528C33E45BBFEC6CFE851095D/@KeyIndexDefined">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Scope/=E8F0594528C33E45BBFEC6CFE851095D/Type/@EntryValue">InCSharpProjectFile</s:String>
+	<s:Boolean x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/ShortenQualifiedReferences/@EntryValue">True</s:Boolean>
+	<s:String x:Key="/Default/PatternsAndTemplates/LiveTemplates/Template/=67C1E0A08AB96D4BB8249F5551FDFD07/Text/@EntryValue">// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.&#xD;// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.&#xD;&#xD;namespace PInvoke&#xD;{&#xD;    /// &lt;content&gt;&#xD;    /// Contains the &lt;see cref="$NESTED$"/&gt; nested type.&#xD;    /// &lt;/content&gt;&#xD;	public partial class $BASECLASS$&#xD;	{&#xD;		/// &lt;summary&gt;&#xD;		/// &lt;/summary&gt;&#xD;		public class $NESTED$&#xD;		{&#xD;			$END$&#xD;		}&#xD;	}&#xD;}</s:String></wpf:ResourceDictionary>
```

Comment:
All this changes are adding a new R# template to create nested classes as we have everywhere.

Don't know why R# choose the WPF resource format for their storage but it's unreadable in git...